### PR TITLE
Add punkts to the sentences

### PR DESCRIPTION
**Pull request by Rahaf91**

### CODE REVIEW

**Review by AbdulrahmanTabbakh — APPROVED** [review]

File: html/html.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Test
+
+## Testing the CLI
+
+### What is Important to Remember:
+
+-`Do not add nested git repository into another git repositroy`
```
Comment (ahohnsen) [inline html/html.md:7]: Tiny typo :) 
```suggestion
-`Do not add nested git repository into another git repository`
```

**Review by laurasliwa — APPROVED** [review]
Really good!

**Comment by Sannsfilm** [comment]
**well done @Rahaf91 ! sehr schön**

**Comment by IsabellUrban** [comment]
Looks great 😃

File: html/html.md (hunk 1 above)
Comment (Rahaf91) [reply html/html.md]: done

**Pull request merged**